src/test/scala/uk/gov/hmrc/test/ui/cucumber/stepdefs/CommonStepDef.scala

```diff
@@ -204,9 +205,20 @@ class CommonStepDef extends BaseStepDef {
 
   And("""^I fill section6""") { () =>
     TransportLeavingTheBorderPage.fillPage("Sea transport"); continue()
-    SupervisingCustomsOfficePage.fillPage("GBBTH001"); continue()
+    if(isClearance){
+      WarehousePage.fillPage("yes", "R1234567GB");
+    }
+    if(isSupervisingCustomsOfficePageVisiblePC || isClearance){
+      SupervisingCustomsOfficePage.fillPage("GBBTH001"); continue()
+    }
     InlandOrBorderPage.fillPage("Customs controlled location"); continue()
-    InlandModeOfTransportPage.fillPage("Postal or mail"); continue()
+    if(!isBorderLocation) {
```

Comment:
remove logic here

Reply:
we need this logic for some scenarios, when i run the tests it failed.. will findout, but for the time being i removed this.